library/Vanilla/Web/MasterView.twig

```diff
@@ -30,7 +30,7 @@
                 <header id="titleBar"></header>
                 <div id="app" class="page-minHeight">
                     {%- if seoContent|default(false) -%}
-                        <noscript>
+                        <noscript aria-hidden="true">
```

Comment:
We can't put this here. This exists for users search engines and users without javascript enabled.

It gets entirely replaced if the react app mounts.